### ISSUE TITLE
Extra semicolon produced by mixins

### ISSUE THREAD

**Issue by davidtheclark** [webschik/postcss-less#45]
Tied to https://github.com/stylelint/stylelint/issues/1389

Given the following:

```less
.foo { .thing(); }
```

I end up with a rule node that erroneously has a `;` in `raws.after`. The rule node that is produced when rendered to a string (`node.toString()`) looks like this: `.foo { .thing(){}; }`. Looking at that, it's clear that the semicolon is superfluous.

**Comment by webschik** [comment]
Thanks, @davidtheclark ! 

I will check it asap.

**Comment by kachkaev** [comment]
Hi @webschik,

Any updates on this by a chance? Looks like this bug causes a problem in atom-linter (see the referenced issues above).

**Comment by webschik** [comment]
Hi @kachkaev,

sorry, I didn't have a time for the task. I will try to resolve this issue this week.

**Comment by webschik** [comment]
Hi guys,

Probably I don't understand the problem: if you don't want to have an extra semicolon, just write your LESS in correct way:

````less
.foo { .thing() }
````

and `postcss-less` returns this result from `toString()` without extra semicolon. But if somebody wants always have an extra semicolons, he can write

````less
.foo { .thing(); }
````

and `postcss-less` also return the correct output as input.

I don't think that it's a good idea if `postcss-less` will be reformat the input code.

**Comment by kachkaev** [comment]
Hi @webschik,

The problem that I see here is that the current behaviour produces wrong error positions during linting. These have to be precise for any input code and now it is not the case. How would you recommend solving [stylelint/stylelint#1409](https://github.com/stylelint/stylelint/issues/1409)?

**Comment by webschik** [comment]
@kachkaev, I see. I will check why the positions are incorrect.

**Comment by webschik** [comment]
I've fixed the bug with positioning and released a version 0.13.0.

Now for code

````less
.class {
  .mixin2;
}
````

an error is on line 2, column 10
